File: src/main/java/com/thebluehats/server/core/modules/RomanNumeralConverterModule.java
package com.thebluehats.server.core.modules;

import com.google.inject.AbstractModule;
import com.google.inject.Provides;
import com.google.inject.Singleton;
import com.thebluehats.server.game.utils.RomanNumeralConverter;

public class RomanNumeralConverterModule extends AbstractModule {
    @Provides
    @Singleton
    static RomanNumeralConverter provideRomanNumeralConverter() {
        return new RomanNumeralConverter();
    }

    @Override
    protected void configure() {
    }
}

File: src/main/java/com/thebluehats/server/core/modules/RomanNumeralConverterModule.kt
package com.thebluehats.server.core.modules

import com.google.inject.AbstractModule
import com.google.inject.Provides
import com.google.inject.Singleton
import com.thebluehats.server.game.utils.RomanNumeralConverter

class RomanNumeralConverterModule : AbstractModule() {
    override fun configure() {}

    companion object {
        @Provides
        @Singleton
        fun provideRomanNumeralConverter(): RomanNumeralConverter {
            return RomanNumeralConverter()
        }
    }
}